pkg/config/validate.go

```diff
@@ -175,6 +179,9 @@ func validateModule(p modulePath, m Module, bp Blueprint) error {
 	if m.ID == "" {
 		errs.At(p.ID, fmt.Errorf(errorMessages["emptyID"]))
 	}
+	if invalidModuleID[m.ID] {
+		errs.At(p.ID, fmt.Errorf(errorMessages["invalidID"]))
```

Comment:
* let's keep it simple `if m.ID == "vars"`, no need for set;
* there is no formatting happening , use `errors.New` instead of `fmt.Errorf`;
* `errorMessages` was a bad design decision made in the past, please don't use it, string literal will suffice;
* The error message can be more descriptive, e.g. `can not use "vars" as a module id`

Reply:
Updated.